.github/workflows/release-test.yml

```diff
@@ -0,0 +1,199 @@
+name: Release tests
+
+on:
+  schedule:
+    - cron: '0 3 * * 6'
+  push:
+    # Run on all new release candidates and release tags
+    tags:
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-RC[0-9]*'
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]'
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9].*'
+  workflow_dispatch:
+    inputs:
+      riot_version:
+        description: 'RIOT version to checkout'
+        required: true
+        default: 'master'
+      release_specs_version:
+        description: 'Release-Specs version to checkout'
+        required: true
+        default: 'master'
+      docker_version:
+        description: 'riot/riotbuild docker image version'
+        required: true
+        default: 'latest'
+
+env:
+  DOCKER_MAKE_ARGS: -j
+# split up native and IoT-LAB tasks to parallelize somewhat and prevent
+# to hit Github Limit of 6h per job, can't use matrix because there seems to be
+# some sync happening and the longer running job is killed.
+jobs:
+  native-tasks:
+    runs-on: ubuntu-latest
+    timeout-minutes: 360
+    steps:
+    - name: Generate .riotgithubtoken
+      run: |
+        echo '${{ secrets.RIOT_CI_ACCESS_TOKEN }}' > ~/.riotgithubtoken
+    - name: Checkout Release-Specs
+      uses: actions/checkout@v2
+      with:
+        # TODO: change to RIOT-OS/Release-Specs
+        repository: miri64/Release-Specs
+        path: Release-Specs
+        fetch-depth: 1
+        ref: ${{ github.event.inputs.release_specs_version }}
+    - name: Checkout RIOT
+      uses: actions/checkout@v2
+      with:
+        repository: RIOT-OS/RIOT
+        path: RIOT
+        fetch-depth: 1
+        ref: ${{ github.event.inputs.riot_version }}
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.x
+    - name: Install Python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox tox-gh-actions junit2html
```

Comment:
I think the tox-gh-actions dependency is not needed. I checked the tox.ini file in the release specs and I also think the gh-actions env is not necessary. (and I also think it's not necessary for riotctrl)

Reply:
Oh yeah, this was an accidental leftover from way back when. Will remove.

Reply:
I've started another test run over at https://github.com/miri64/RIOT/runs/949582657. This should show us in a moment if it works without `tox-gh-actions`.

Reply:
It works without the dependency so the gh-actions env is probably not needed indeed.

Reply:
[Exihibit A](https://github.com/miri64/RIOT/runs/949582657?check_suite_focus=true#step:11:33) and [exihibit B](https://github.com/miri64/RIOT/runs/949582602?check_suite_focus=true#step:9:37).